src/BigBlueButton.php

```diff
@@ -587,4 +587,9 @@ private function requestUrl(string $url, string $payload = '', string $contentTy
 
         return $response->getBody();
     }
+
+    public function buildUrl(string $method = '', string $params = '', bool $append = true): string
```

Comment:
If we allow developers to create a custom url, should we not also allow them to use the other internal functions like requestUrl, processXMLRespose, etc. ?

Reply:
Maybe yes. But: This PR only pulls and merges upstream. I would strictly distinguish our changes from syncing in upstream.

Reply:
Personally, I would move the URL building and response processing out of the BBB class and create own "services" to prevent mixing up too much public methods. But this would be a bigger refactoring.

The reason why I would takeover this change for now, is that we should ease consumers of the original library to use our library instead by providing the same features.